.github/workflows/label-conflicting-pr.yml

```diff
@@ -3,12 +3,13 @@ on:
   push:
   pull_request_target:
     types: [synchronize]
-    
+
 permissions:
   pull-requests: write
 
 jobs:
   main:
+    if: github.event.pull_request.user.login != 'dependabot[bot]'
```

Comment:
(For my edification: the `'` here is part of the value interpreted by GH, not a YAML metacharacter.)